aimmo-game/simulation/world_map.py

```diff
@@ -97,6 +97,7 @@ def all_cells(self):
         return self.grid.values()
 
     def score_cells(self):
+        # TODO: consider changing this tuple might too slow
```

Comment:
nnnnnniiiiiiyyyyy

Reply:
Wooooooooops! #368. Deleting the comment now, again XD